scripts/gen_help_html.lua

```diff
@@ -1123,9 +1123,8 @@ function M._test()
       'if "expected" is given, "actual" is also required'
     )
     if expected then
-      return assert(
-        cond,
-        ('expected %s, got: %s'):format(vim.inspect(expected), vim.inspect(actual))
+      return (
+        assert(cond, ('expected %s, got: %s'):format(vim.inspect(expected), vim.inspect(actual)))
```

Comment:
For better readability, we can do like:
```
assert(cond, "<ERROR MSG>")
return cond
```

Reply:
+1, the extra parens are subtle, easy to miss, and non-obvious.